src/entity/entity.js

```diff
@@ -225,6 +247,7 @@ class EntityManager {
 
         // Load the sprite into the engine and store the ptr
         entity.sprite = this.assetManager.loadSprite(entity.spriteName);
+        entity.sprite.anchor.set(0.5); // Can't be called in entity constructor because sprite isn't loaded yet
```

Comment:
I don't like doing this categorically to all entities here, but I can't reference an entity's sprite until after the entity manager has `load`ed the sprite, so here it is. This will set the origin for a sprite to the center of the image rather than the top left corner, which I needed for rotation (and I think may be more intuitive overall -- thoughts?)

Reply:
Nope, this is good! The load function will apply and engine level inits. Otherwise load() in Entity class works too since it is called right after.